docs/encryption.md

```diff
@@ -8,7 +8,8 @@ magic supports the following encryption functions in its core API:
 * [magic.PwdEncryptStream | magic.PwdDecryptStream](#magicpwdencryptstream--magicpwddecryptstream): Symmetric encryption for streams using a password
 
 The alt API also supports the following functions:
-* [magic.alt.encrypt.AES\_{128,192,256}\_CBC\_HMAC\_SHA{256,384,512} | magic.alt.decrypt.AES\_{128,192,256}\_CBC\_HMAC\_SHA{256,384,512}](magicaltencryptaes_128192256_cbc_hmac_sha256384512--magicaltdecryptaes_128192256_cbc_hmac_sha256384512): Implements AES encryption with different key lengths and SHA2 algorithms
+* [magic.alt.encrypt.AES_{128,192,256}\_GCM | magic.alt.decrypt.AES\_{128,192,256}\_GCM](#magicaltencryptaes_128192256_gcm--magicaltdecryptaes_128192256_gcm): Implements AES-GCM authenticated encryption with different key lengths. FIPS 140-2 approved.
+* [magic.alt.encrypt.AES\_{128,192,256}\_CBC\_HMAC\_SHA{256,384,512} | magic.alt.decrypt.AES\_{128,192,256}\_CBC\_HMAC\_SHA{256,384,512}](#magicaltencryptaes_128192256_cbc_hmac_sha256384512--magicaltdecryptaes_128192256_cbc_hmac_sha256384512): Implements AES encryption with different key lengths and SHA2 algorithms
```

Comment:
Should the 2 lines be similar? Above you mention `AES-GCM ...` but in here it's `AES`. Nothing major, just to have a similar doc style.

Reply:
Good catch, updated. Could you re-approve?